cleanlab_studio/internal/api/api.py

```diff
@@ -379,3 +380,21 @@ def get_prediction_status(api_key: str, query_id: str) -> Dict[str, str]:
     handle_api_error(res)
 
     return cast(Dict[str, str], res.json())
+
+
+def get_tlm_confidence(
+    api_key: str,
+    prompt: str,
+) -> JSONDict:
+    """
+    Prompt Trustworthy LLM with a question, and get back its answer along with a confidence score
+    :param api_key:
+    :return: a JSON dict in the format
+        {'answer': {tlm generated answer},
```

Comment:
JSON strings are always double-quoted. Also, nit: I'd suggest a different syntax for "template variables" because they conflict with braces for JSON objects. You can also convey types more effectively.

perhaps

```json
{"answer": "<TLM-generated answer>",
 "confidence_score": <score between 0 and 1>}
```

Reply:
fixing. sorry, should've paid more attention copying this over